e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/CreateNodeIntegrationTest.java

```diff
@@ -118,6 +120,8 @@ void testCreateFile()
                 .extract().response();
 
         Assertions.assertEquals(200, s3Response.statusCode());
+        String stringS3response = s3Response.asString();
+        Assertions.assertTrue(stringS3response.contains("Contents"));
```

Comment:
I can't tell from reading the test what "Contents" is meant to be. Is it the text content of a file? Is it something that Localstack puts in the S3 response? Is it part of the `toString` representation of a `Response` object?

Reply:
"Contents" is the tag of S3 bucket (Localstack) xml structure 
![Screenshot 2024-04-18 at 15 45 01](https://github.com/Alfresco/hxinsight-connector/assets/122363592/6be2d652-d198-442a-b5f2-82f7f1afee7a)